src/main/java/com/dev/cinema/model/dto/request/UserRequestDto.java

```diff
@@ -15,6 +16,7 @@
     private String email;
     private String password;
     private String repeatPassword;
+    private Role role;
```

Comment:
it is redundant to return roles with user in the dto

Reply:
done